skrub/_reporting/_data/templates/column-associations.html

```diff
@@ -1,5 +1,14 @@
 <article class="margin-t-s flex-reverse wrapper gap-l">
-    {% if summary["top_associations"] %}
+    {% if summary["associations_skipped"] %}
+
+    <div id="skipped-associations-alert" class="alert-info-dismissable flex space-between">
```

Comment:
When is it the case, that we could see this div when interacting as a user? Would I need to specifically request the HTML page?

Reply:
whenever associations are skipped because of `max_associations_columns` and the report is generated outside of an expression's html_repr

Reply:
Oh yes, I misread the cause. I thought it was when we completely skipped.